clustered/server/src/main/java/org/ehcache/clustered/server/state/ClientMessageTracker.java

```diff
@@ -73,4 +73,11 @@ public boolean isConfigureApplicable(UUID clientId, long timestamp) {
     return true;
   }
 
+  public UUID getConfiguredEntityId() {
```

Comment:
This change is not required because this is not set by Active, only a newly promoted active needs this info.